packages/js/product-editor/src/components/details-summary-block/block.json

```diff
@@ -0,0 +1,30 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-summary",
+	"title": "Product summary",
+	"category": "widgets",
+	"description": "The product summary.",
+	"keywords": [ "products", "summary", "excerpt" ],
+	"textdomain": "default",
+	"attributes": {
+		"align": {
+			"type": "string"
+		},
+		"direction": {
+			"type": "string",
+			"enum": [ "ltr", "rtl" ]
+		},
+		"label": {
+			"type": "string"
+		}
+	},
+	"supports": {
+		"align": false,
+		"html": false,
```

Comment:
It would be nice if we could enable the `html` flag, to allow users to edit as html as well.
But unfortunately the `HtmlBlock` doesn't work with our reliance on the core editor and edits the blocks value only.

I wonder if it would be easy to add this? Or create a separate `supports` that would allow for this.